config/routes.rb

```diff
@@ -33,6 +33,11 @@
           resources :submissions, param: :version, only: [:index] do
             get "feedback"
           end
+
+          resources :scores, only: [:index, :show],
+                    param: :email, :constraints => { :email => /[^\/]+/ }
+
+          put "scores/:email", :constraints => { :email => /[^\/]+/ }, to: "scores#update_latest"
```

Comment:
I don't think we should repurpose 
scores:/email to update latest, but perhaps create something like scores/:email/update-latest. 

Because if we are putting scores:/email, we should be going with the default update behavior, not update_latest.